components/HeaderDropdownNav.js

```diff
@@ -0,0 +1,51 @@
+import { Component } from "react";
+
+import { Dropdown } from "react-bootstrap";
+
+const HeaderDropdownItem = props => (
+  <Dropdown.Item className="header-nav-item-dropdown" onClick={props.onClick}>
+    {props.title}
+  </Dropdown.Item>
+);
+
+class HeaderDropdownNav extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      network: "Testnet One"
+    };
+
+    this.selectNetwork = this.selectNetwork.bind(this);
```

Comment:
It seems easier to define the methods with arrow functions and avoid this manual rebinding. So, you just drop this line and define the method as:

```
selectNetwork = (e) => {
  ...
}
```

Just a random article on this topic: https://javascriptweblog.wordpress.com/2015/11/02/of-classes-and-arrow-functions-a-cautionary-tale/

Reply:
Thanks. Switched to that.